js/cli/package.json

```diff
@@ -36,7 +37,7 @@
     "@project-serum/anchor": "^0.24.2",
     "@raindrop-studios/sol-command": "^0.4.0",
     "@raindrop-studios/sol-kit": "^0.1.78",
-    "@raindrops-protocol/raindrops": "^0.0.7",
+    "@raindrops-protocol/raindrops": "workspace:js/lib",
```

Comment:
I took a quick peek at the docs for workspace resolution and yarn is still listing this as "experimental syntax" - recommended is to use `workspace:*`
https://yarnpkg.com/features/workspaces#workspace-ranges-workspace

Tested the lib locally with yarn link and it seems to work well. Super quick to build with yarn 3 too!